app/models/UserModel.php

```diff
@@ -0,0 +1,67 @@
+<?php
+namespace app\models;
+use app\core\Database;
+
+class UserModel extends Database
```

Comment:
Noting that this model is a little different from the others in that it extends the `Database` class which the others inject it via the construct function.  I'd prefer a consistent approach here unless there's a good reason not to.

Reply:
Agree. I have been learning and revising my code base, thus creating a bit of un-consistency.